fuzz/README.md

```diff
@@ -0,0 +1,109 @@
+# Fuzzing
+
+`simplicity` has a fuzzing harness setup for use with honggfuzz.
+
+To run the fuzz-tests as in CI -- briefly fuzzing every target -- simply
+run
+
+    ./fuzz.sh
+
+in this directory.
+
+The script builds honggfuzz as part of its startup.
+
+To build honggfuzz you must have libunwind on your system, as well as
+libopcodes and libbfd from binutils **2.38** on your system. The most
+recently-released binutils 2.39 has changed their API in a breaking way.
+
+On Nix, you can use the provided nix shell
+
+    nix-shell
+
+Then run fuzz.sh as above.
+
+# Long-term fuzzing
+
+To see the full list of targets, the most straightforward way is to run
+
+    source ./fuzz-util.sh
+    listTargetNames
+
+To run each target for an hour, run
+
+    ./cycle.sh
+
+This script uses the `chrt` utility to try to reduce the priority of the
+jobs. If you would like to run for longer, the most straightforward way
+is to edit `cycle.sh` before starting. To run the fuzz-tests in parallel,
+you will need to implement a custom harness.
+
+To manually run a single target indefinitely, run
+
+    cargo hfuzz run <target>
+
+You need to install cargo-honggfuzz
```

Comment:
In d4852ef27c8720437f05bc10a4d84e04289a71d8:

I'd say

```
You may need to install cargo-honggfuzz:

    cargo install honggfuzz --no-default-features

though it may already be installed. It is installed by the shell-scripts and is available in the provided nix-shell.

If you have linker errors see the above information about libbfd and libopcodes.
```